### DISCUSSION TITLE
"Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

### DISCUSSION THREAD

**Discussion by rendomnet** [community/community#51545] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have paid for Copilot but cant activate it because it gives me this error message!

`GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."`

**Comment by Akash1134** [comment]
Hi @rendomnet ,

I believe the message you're getting is only a warning that Technical Previews for Copilot can no longer be extended.

- Clicking outside the warning, please check if you have access to Copilot. Do you have another account you might be logged in with? - - Can you click on the Accounts icon at the bottom left and check that you're logged in with the correct account?

Also, sign out of Copilot in your IDE and log back in to see the changes above.  Thanks!
